.gitlab-ci.yml

```diff
@@ -435,11 +435,37 @@ check-polkadot-companion-build:
     - job:                         test-linux-stable-int
       artifacts:                   false
   script:
-    - ./.maintain/gitlab/check_polkadot_companion_build.sh
+    - ./.maintain/gitlab/check_companion_build.sh paritytech polkadot
   after_script:
     - cd polkadot && git rev-parse --abbrev-ref HEAD
   allow_failure:                   true
 
+check-cumulus-companion-build:
+  stage:                           build
+  <<:                              *docker-env
+  <<:                              *test-refs-no-trigger
+  needs:
+    - job:                         test-linux-stable-int
```

Comment:
this is the longest job, what's the reason to depend on it? Similarly, you could skip this block entirely (to avoid unnecessary complication) and the next stage won't start if the previous is not successful.

Reply:
I don't know the reason - polkadot-check-companion-build also relies on it so I simply followed that. I suppose there's no reason that should be the case. I'll try chucking these jobs into the test phase and see what there difference in pipeline run-length is.